ci/migrate/setup.sh

```diff
@@ -31,7 +31,7 @@ kubectl label nodes --all nodetype=storage
 
 echo "Create application with cStor volume on SPC"
 
-bdname=$(kubectl -n openebs get blockdevices -o jsonpath='{.items[*].metadata.name}')
+bdname=$(kubectl -n openebs get blockdevices -o jsonpath='{.items[?(@.spec.details.deviceType=="sparse")].metadata.name}')
```

Comment:
Are we testing using only sparse devices?

Reply:
Yes only 1 sparse disk is used for integration test